docs/resources/compute_instance.md

```diff
@@ -194,10 +194,10 @@ The following arguments are supported:
 
 * `name` - (Required, String) A unique name for the resource.
 
-* `image_id` - (Optional, String) Required if `image_name` is empty. The image ID of
+* `image_id` - (Optional, String, ForceNew) Required if `image_name` is empty. The image ID of
```

Comment:
This will rebuild the server with a new OS instead of ForceNew. The instance id will not change.

Reply:
The is a 'ForceNew: true' in the code. So I think there should also be a 'ForceNew' in the doc.